implement-shell-tools/wc/wc.py

```diff
@@ -0,0 +1,72 @@
+import argparse
+import os 
+import re
+
+parser = argparse.ArgumentParser(
+                    prog='wc command',
+                    description='word, line, character, and byte count')
+
+parser.add_argument("-c", "--bytes", help="The number of bytes in each input file is written to the standard output.", default = None, action="store_true")
+parser.add_argument("-l", "--lines", help="The number of lines in each input file is written to the standard output.", default = None, action="store_true")
+parser.add_argument("-w", "--words", help="The number of words in each input file is written to the standard output.", default = None, action="store_true")
+parser.add_argument("paths", nargs="+", help="The file to search")
+
+args = parser.parse_args()
+cwd = os.getcwd()
+
+
+def count_lines(content):
+    return len(content.splitlines())
+    
+def count_words(content):
+    return len(re.findall("[a-zA-Z\-\.'/]+", content))
+
+def count_bytes(content):
+    return len(content.encode('utf-8'))
+
+def process_file_content(content, path):
+    if not args.lines and not args.words and not args.bytes:
+        global no_options
+        no_options = True
+        lines_in_file = count_lines(content)
+        words_in_file = count_words(content)
+        bytes_in_file = count_bytes(content)
+        print(lines_in_file, words_in_file, bytes_in_file, path)
+        return {'lines': lines_in_file, 'words':words_in_file, 'bytes':bytes_in_file}
+    output = {}
+    if args.lines:
+        output['lines'] = count_lines(content)
+    if args.words:
+        output['words'] = count_words(content)
+    if args.bytes:
+        output['bytes'] = count_bytes(content)
+    print(' '.join([ str(val) for val in output.values()]), path)
+    return output
+
+total_lines, total_words, total_bytes = 0, 0, 0
+for path in args.paths:
+    try:
+        file_path_to_procccess = os.path.join(cwd, path)
```

Comment:
```suggestion
        file_path_to_proccess = os.path.join(cwd, path)
```
Typo: There was an extra c :D

This is not relevant for the code working, but in a real code review it's likely to get pointed out, so I do it here as well. If you decide to change it, note that you'll have to change every place where that variable is used.